src/components/visualizations/styles/_variables.scss

```diff
@@ -1,8 +1,9 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
 @import "~@gooddata/goodstrap/lib/core/styles/variables";
 
 $insight-small-only: "only screen and (max-width: 480px)";
 $responsive-legend: "only screen and (max-width: 767px)";
 
 $footer-border: #e2e8ed;
 $footer-border-top: $gd-color-disabled;
+$light-gray-color: #6d7680;
```

Comment:
Did you try to use the existing one?
https://github.com/gooddata/gdc-goodstrap/blob/4080ccbe7210b4f9affcf1a8cfe0e690c90f4d0b/src/core/styles/variables.scss#L20